docs/plat/nvidia-tegra.md

```diff
@@ -26,7 +26,7 @@ without changing any makefiles.
 Preparing the BL31 image to run on Tegra SoCs
 ===================================================
 CROSS_COMPILE=<path-to-aarch64-gcc>/bin/aarch64-none-elf- make PLAT=tegra \
```

Comment:
This style doesn't work in markdown, the <> are interpreted as tags of some kind and the content is eliminated. You should blockquote this whole paragraph using ``` before and after, e.g.

```
CROSS_COMPILE=<path-to-aarch64-gcc>/bin/aarch64-none-elf- make PLAT=tegra \
TARGET_SOC=<target-soc e.g. t210|t132> BL32=<path-to-trusted-os-binary> \
SPD=<dispatcher e.g. tlkd> all
```